CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(astra_camera)
 if(NOT WIN32)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wextra")
```

Comment:
If we want to stay close to the ROS1 package, should we keep CMake and c++ standard versions the same as the original package ? Otherwise we may start using features that ROS1 doesn't support